ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -76,6 +76,8 @@
 {% include 'eos/radius-servers.j2' %}
 {# router l2-vpn#}
 {% include 'eos/router-l2-vpn.j2' %}
+{# router msdp #}
```

Comment:
Please ensure this is the place it is rendered in EOS. I have not tested, so just resolve if it is ok.